specification/security/resource-manager/readme.md

```diff
@@ -68,6 +68,10 @@ directive:
     where: $.paths
     from: applications.json
     reason: Suppression of OperationsAPI as it doesn't apply to this specific file.
+  - suppress: OperationsAPIImplementation
```

Comment:
why are you suppressing the operations API?

Reply:
no idea, it was changed by the auto tool that generated the pr, i will change it